pkg/project/project.go

```diff
@@ -80,7 +81,7 @@ func (p *Project) CommonSetupSteps() error {
 	p.logger.Log("info", "executing common setup steps")
 	steps := []Step{
 		Step{
-			Run: "kubectl config use-context minikube",
+			Run: "kubectl config use-context " + p.cfg.K8sContext,
```

Comment:
When we are here I would rather prefer to run all commands with `--context` flag instead.

Reply:
I cannot say if it is used also somewhere else or if some other part is relying upon

Reply:
I think this comment https://github.com/giantswarm/e2e-harness/pull/101#discussion_r205736077 belongs here. Yes this is out of scope and definitely not a blocker here.

Reply:
I would also rather prefer adding a flag to the `e2e-harness` command to accept the context to use. This is way more flexible and transparent, instead of some magic we hide internally and everybody just has to know.